src/util/campaignUtils.js

```diff
@@ -120,6 +120,7 @@ export function getPromoFromBasket(promoFundId, apollo) {
 		query: promoCampaign,
 		variables: {
 			promoFundId: String(promoFundId)
-		}
+		},
+		fetchPolicy: 'network-only',
```

Comment:
Ensures we update if there is prior bonus credit found in the initial header query.